A0Auth0.podspec

```
@@ -17,7 +17,7 @@ Pod::Spec.new do |s|
   s.requires_arc = true
 
   s.dependency 'React-Core'
-  s.dependency 'Auth0', '2.7.2'
-  s.dependency 'JWTDecode', '3.1.0'
-  s.dependency 'SimpleKeychain', '1.1.0'
+  s.dependency 'Auth0', '2.10'
+  s.dependency 'JWTDecode', '3.2.0'
+  s.dependency 'SimpleKeychain', '1.2.0'
```

```suggestion
  s.dependency 'Auth0', '2.10'
  s.dependency 'JWTDecode', '3.3.0'
  s.dependency 'SimpleKeychain', '1.3.0'
```

New versions of JWTDecode.swift and SimpleKeychain have been released.

---

We should update the Podfile.lock as well.

---

we can't update that as Auth0.swift is also using the same version. We have to first update the dependency in Auth0.swift first

---

That's right, my bad.

---

I haven't released the new version yet.